web/main.go

```diff
@@ -73,7 +75,13 @@ func main() {
 		}
 	}
 
-	server := srv.NewServer(*addr, *grafanaAddr, *templateDir, *staticDir, uuid, *controllerNamespace, clusterDomain, *reload, client, k8sAPI)
+	reHost, err := regexp.Compile(*enforcedHost)
```

Comment:
is the empty string a valid regex?  does it match everything?  I would expect omitting the --enforced-host flag to effectively disable this protection.

Reply:
Yes the empty string matches everything. So omitting `--enforced-host` makes `enforcedHost` default to the empty string, which allows any `Host` header.